### PR TITLE
Bug 1508395 - upgrade tc-lib-validate

### PR DESCRIPTION
**Pull request by djmitche**
..and add .gitignore for node_modules/

This should only affect writeDocs

### CODE REVIEW

**Comment by jhford** [comment]
Let's hold off on this until #3 merges

**Comment by djmitche** [comment]
@jhford #3 is merged now -- will you accept this if I rebase?

**Comment by djmitche** [comment]
(rebased)

**Review by jhford — APPROVED** [review]

**Pull request merged**
